keps/sig-storage/20191031-storage-capacity-constraints-for-pod-scheduling.md

```diff
@@ -204,7 +206,7 @@ class and its parameters.
 In contrast to local storage, network attached storage can be made
 available on more than just one node. However, for technical reasons
 (high-speed network for data transfer inside a single data center) or
-political reasons (data must only be stored and processed in a single
+regularity reasons (data must only be stored and processed in a single
```

Comment:
I think Abdullah meant "regulation"

Reply:
So did I, I just couldn't type it... Fixed.